test/integration/patients/patient/patient-flow.js

```diff
@@ -1099,41 +1098,63 @@ context('patient flow page', function() {
         name: 'Socket Action',
       },
       relationships: {
-        'flow': getRelationship(testFlow),
+        flow: getRelationship(testFlow),
+        state: getRelationship(stateTodo),
       },
     });
 
     cy
       .routeAction(fx => {
         fx.data = otherAction;
 
+        fx.included.push(testFlow);
+
         return fx;
       });
 
     cy.sendWs({
       category: 'ResourceCreated',
```

Comment:
I'm thinking about moving this down to the `'socket notifications'` test at the bottom of this file.

Reply:
I'll leave it here for now.